exercises/complex-numbers/test/test_complex_numbers.c

```diff
@@ -0,0 +1,192 @@
+    Complex cases[][3] = {
+        { { .real = 1.0, .imag = 0.0 }, { .real = 2.0, .imag = 0.0 }, { .real = 0.5, .imag = 0.0 } },
+        { { .real = 0.0, .imag = 1.0 }, { .real = 0.0, .imag = 2.0 }, { .real = 0.5, .imag = 0.0 } },
+        { { .real = 1.0, .imag = 2.0 }, { .real = 3.0, .imag = 4.0 }, { .real = 0.44, .imag = 0.08 } }
+    };
+    
+    unsigned int i;
+    for (i = 0; i < sizeof(cases) / sizeof(Complex[3]); i++) {
+        Complex result = c_div(&cases[i][0], &cases[i][1]);
+
+        TEST_ASSERT_EQUAL(result.real, cases[i][2].real);
+        TEST_ASSERT_EQUAL(result.imag, cases[i][2].imag); 
+    }
+}
+
+// absolute value
+void test_absolute_value(void) {
+    Complex cases[] = {
+        { .real = 5.0, .imag = 0.0 },
+        { .real = -5.0, .imag = 0.0 },
+        { .real = 0.0, .imag = 5.0 },
+        { .real = 0.0, .imag = -5.0 },
+        { .real = 3.0, .imag = 4.0 }
+    };
+
+    unsigned int i;
+    for (i = 0; i < sizeof(cases) / sizeof(Complex); i++) {
+        double result = c_abs(&cases[i]);
+
+        TEST_ASSERT_EQUAL(5.0, result);
+    }
+}
+
+// conjugate
+void test_conjugate(void) {
+    Complex cases[][2] = {
+        { { .real = 5.0, .imag = 0.0 }, { .real = 5.0, .imag = 0.0 } },
+        { { .real = 0.0, .imag = 5.0 }, { .real = 0.0, .imag = -5.0 } },
+        { { .real = 1.0, .imag = 1.0 }, { .real = 1.0, .imag = -1.0 } }
+    };
+
+    unsigned int i;
+    for (i = 0; i < sizeof(cases) / sizeof(Complex[2]); i++) {
+        Complex result = c_conjugate(&cases[i][0]);
+
+        TEST_ASSERT_EQUAL(result.real, cases[i][1].real);
+        TEST_ASSERT_EQUAL(result.imag, cases[i][1].imag);
+    }
+}
+
+// real part
+void test_real_part(void) {
+    Complex cases[] = {
+        { .real = 1.0, .imag = 0.0 },
+        { .real = 0.0, .imag = 1.0 },
+        { .real = 1.0, .imag = 2.0 }
+    };
+
+    unsigned int i;
+    for (i = 0; i < sizeof(cases) / sizeof(Complex); i++) {
+        TEST_ASSERT_EQUAL(c_real(&cases[i]), cases[i].real);
+    }
+}
+
+// imaginary part
+void test_imag_part(void) {
+    Complex cases[] = {
+        { .real = 1.0, .imag = 0.0 },
+        { .real = 0.0, .imag = 1.0 },
+        { .real = 1.0, .imag = 2.0 }
+    };
+
+    unsigned int i;
+    for (i = 0; i < sizeof(cases) / sizeof(Complex); i++) {
+        TEST_ASSERT_EQUAL(c_imag(&cases[i]), cases[i].imag);
+    }
+}
+
+// complex exponential
+void test_exponential(void) {
+    Complex cases[][2] = {
+        { { .real = 0.0, .imag = PI }, { .real = -1.0, .imag = 0.0 } },
+        { { .real = 0.0, .imag = 0.0 }, { .real = 1.0, .imag = 0.0 } },
+        { { .real = 1.0, .imag = 0.0 }, { .real = E, .imag = 0.0 } }
+    };
+
+    unsigned int i;
+    for (i = 0; i < sizeof(cases) / sizeof(Complex[2]); i++) {
+        Complex result = c_exp(&cases[i][0]);
+
+        TEST_ASSERT_EQUAL(result.real, cases[i][1].real);
+        TEST_ASSERT_EQUAL(result.imag, cases[i][1].imag);
+    }
+}
+
+int main(void)
+{
+   UnityBegin("complex_numbers.c");
```

Comment:
@wolf99 @ryanplusplus Did we ever standardize on what to actually use here?

Reply:
This should be the test file name - every other usage so far is `UnityBegin("test/test_complex_numbers.c")`. The [examples given in the Unity repo](https://github.com/ThrowTheSwitch/Unity/blob/master/examples/example_1/test/test_runners/TestProductionCode_Runner.c) indicate that this is correct

Reply:
This is actually shown in the example in https://github.com/exercism/c/blob/master/docs/C_STYLE_GUIDE.md